.github/workflows/test.yml

```
@@ -10,28 +10,17 @@ name: Tests
     - cron: '0 6 * * 0'
 
 jobs:
-  test:
-    name: 'Node.js v${{ matrix.node }}'
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node:
-          - 14
-          - 16
-          - 18
-    steps:
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '${{ matrix.node }}'
-      - uses: actions/checkout@v2
-      - name: 'Cache node_modules'
-        uses: actions/cache@v2
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-node-v${{ matrix.node }}-${{ hashFiles('**/package.json') }}
-          restore-keys: |
-            ${{ runner.os }}-node-v${{ matrix.node }}-
-      - name: Install Dependencies
-        run: npm install
-      - name: Run All Node.js Tests
-        run: npm run test
+  test-node-v14:
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      node-version: 14
+      
+  test-node-v16:
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      node-version: 16
+
+  test-node-v18:
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      node-version: 18
```

[Style] Nit, there was a newline before:

```suggestion
      node-version: 18

```